cfn-resources/project/cmd/resource/resource.go

```diff
@@ -405,7 +405,7 @@ func getProjectByName(name *string, client *mongodbatlas.Client) (event handler.
 		if res.Response.StatusCode == 401 { // cfn test
 			return handler.ProgressEvent{
 				OperationStatus:  handler.Failed,
-				Message:          "Error while deleting project",
+				Message:          "Error while GetOneProjectByName",
```

Comment:
[nit] Shouldn't these be formatted as human readable?
```suggestion
				Message:          "Error while retrieving project by name",
```